js/bundle/tests/decoder_test.js

```diff
@@ -0,0 +1,52 @@
+const wbn = require('../lib/wbn');
+
+describe('Bundle', () => {
+  const bundleBuffer = (() => {
```

Comment:
I think we should have a pre-generated test bundle file to detect unexpected regressions (or format change).

Reply:
Done.